# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1231916820 18000
#      Wed Jan 14 02:07:00 2009 -0500
# Node ID 738cb6271933f66e816114182784790c1c2db8c8
# Parent  841f8e3370c6b98f819681c1662f4844128c0f35
__go_draw_axes__.m: scale markersize by 1/3 instead of 1/6

diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,10 +1,12 @@
 2009-01-14  John W. Eaton  <jwe@octave.org>
 
+	* plot/__go_draw_axes__.m: Scale markersize by 1/3, not 1/6.
+
 	* io/dlmwrite.m: Use '%c' format for character data.
 
 2009-01-13  John W. Eaton  <jwe@octave.org>
 
 	* general/repmat.m: Handle negative row or column dimension.
 
 	* elfun/lcm.m, general/accumarray.m, general/bicubic.m,
 	general/cellidx.m, general/cplxpair.m, general/dblquad.m,
diff --git a/scripts/plot/__go_draw_axes__.m b/scripts/plot/__go_draw_axes__.m
--- a/scripts/plot/__go_draw_axes__.m
+++ b/scripts/plot/__go_draw_axes__.m
@@ -584,17 +584,17 @@ function __go_draw_axes__ (h, plot_strea
 	     data_idx++;
              is_image_data(data_idx) = false;
              parametric(data_idx) = false;
 	     have_cdata(data_idx) = false;
              titlespec{data_idx} = "title \"\"";
 	     usingclause{data_idx} = sprintf ("record=%d", numel (obj.xdata));
 
 	     if (isfield (obj, "markersize"))
-	       mdat = obj.markersize / 6;
+	       mdat = obj.markersize / 3;
 	     endif
 
              if (isfield (obj, "edgecolor"))
 	       if ((strncmp (obj.edgecolor, "flat", 4)
 		    || strncmp (obj.edgecolor, "interp", 6))
 		   && isfield (obj, "cdata"))
 		 if (ndims (obj.cdata) == 2
 		     && (size (obj.cdata, 2) == nc
@@ -1298,17 +1298,17 @@ function [style, typ, with] = do_linesty
       if (! isempty (pt))
 	style = "points";
       endif
     elseif (! isempty (pt))
       style = "linespoints";
     endif
 
     if (isfield (obj, "markersize"))
-      fprintf (plot_stream, " pointsize %f", obj.markersize / 6);
+      fprintf (plot_stream, " pointsize %f", obj.markersize / 3);
       found_style = true;
     endif
   else
     style = errbars;
     found_style = true;
   endif
 
   if (! found_style)
